docs/get_started.md

```diff
@@ -97,16 +97,20 @@ the creation of a new management pack project.
 
 For complete documentation of the `mp-init` tool including an overview of its output, see the [MP Initialization Tool Documentation](references/mp-init.md).
 
-### Template Project
+### Template Projects
 Every new project creates a file system that has the basic project structure required to develop and build a Management Pack.
 Each file and directory is discussed in depth in the [mp-init](references/mp-init.md) documentation. `app/adapter.py` is the adapter's
-entry point and the best starting point.
+entry point and the best starting point. Users can pick between two entry points when creating their project:
+- `Sample Adapter`: Ideal for first-time users;
+  it comes with a template adapter that collects several objects and metrics from the container the adapter is running.
+  The template adapter has comments throughout its code
+  that explain what the code does and how to customize it for your adapter.
 
-`adapter.py` is a template adapter that collects several objects and metrics from the
-container in which the adapter is running. The template adapter has comments throughout its code that explain what the code does
-and how to customize it for your adapter.
+- `New Adapter`: Ideal for experienced users;
+  it comes without any examples, just the necessary code and comments to implement test connection,
+  collection, adapter definition, and endpoints logic.
 
-The methods inside the adapter template are required, and should be modified to generate a custom
+The methods inside the `adapter.py` are required, and should be modified to generate a custom
 adapter. Each method fulfills a request from the VMware Aria Operations collector, and can be tested individually using
 `mp-test` (covered in [Testing a Management Pack](#testing-a-management-pack)).
```

Comment:
I think we need to modify the structure of this a bit. Previously, the discussion about the template adapter applied to every mp-init output, so it was after the walkthrough. However, we should move the selection part up to a 'step 8' in the "Creating a Management Pack" section and discuss the options there, then move into some of the details of how projects and adapters are setup and work (in a generic way) here.

I'm also unsure if we should market the 'New Adapter' as 'Ideal for experienced users' and the sample adapter as 'Ideal for first time users'. I think it's better to just describe what they are, but in the walkthrough section above make sure we direct the user to select 'Sample Adapter'.

Reply:
I also wouldn't say `necessary code`. Open to ideas here, but maybe something more along the lines of `minimum code` or `minimal code`.